microsoft-edge/extensions/publish/create-dev-account.md

```diff
@@ -6,142 +6,239 @@ ms.author: msedgedevrel
+To use a particular name, that name must be available, and you must have the rights to use it.  For a company account, this must be the registered business name of your organization.
 
 The maximum length for this field is fifty (50) characters.
 
 
 <!-- ------------------------------ -->
-#### Contact details
+#### Contact info
+
+In the **Contact info** section, enter contact information that Microsoft will use to contact you about any account issues:
+
+![Contact info](./create-dev-account-images/contact-info.png)
+
+For a company account, you must use the registered email address that's associated with your organization.
 
-This field contains any contact information that Microsoft uses to contact you about any account issues.  After registration is complete, you receive an email confirmation.  For a company account, you must use the registered email address associated with your organization.
+After registration is complete, you receive a confirmation email.  
 
 
 <!-- ------------------------------ -->
 #### Company approver
 
-For a company account, you must provide the contact information of your company approver.  The contact information includes name, email address, and phone number.  As a part of the verification process, Microsoft contacts the specified company approver, to make sure that your extension belongs to your organization.
+For a company account, you must provide the contact information of your company approver.  If you selected the **Company** option button in the **Account type** section, the **Company approver** section appears:
+
+![Company approver](./create-dev-account-images/company-approver.png)
+
+The contact information includes name, email address, and phone number.  As a part of the account verification process, Microsoft contacts the specified company approver, to verify that you are authorized to create and manage this account on behalf of your organization.
+
+
+<!-- ------------------------------ -->
+#### Developer agreement
+
+1. In the **Developer agreement** section, click the **App Developer Agreement** link, and then read the [Microsoft Store App Developer Agreement](https://go.microsoft.com/fwlink/?linkid=528905).
+
+   ![Developer agreement section and Finish button](./create-dev-account-images/agreement-finish.png)
+
+1. In the **Microsoft Edge Developer Account Registration** form, select the **I accept the terms and conditions** checkbox.
+
+1. Click the **Finish** button.
 
 
 <!-- ====================================================================== -->
-## Next steps
+## Step 3: Resolve any Partner Center errors
+
+After submitting the form, Partner Center might display an error in the **Properties** page, such as "Something went wrong.  Please try again. correlationId : undefined":
```

Comment:
Can we get a better error message? Does the error message really contain the word "undefined"? That seems bad because 1. Partner Center fails and 2. it outputs a very cryptic error message.
But also, the screenshot shows that the user has reached the next step anyway (**Properties**). Is this error really blocking the whole process? Or can the user continue from here?